Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/MembershipsController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Know_Your_Nation_Speedy.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Know_Your_Nation_Speedy.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MembershipsController : ControllerBase
+    {
+        private readonly MyDbContext _db;
+        readonly IConfiguration _config;
+        public MembershipsController(MyDbContext context, IConfiguration config)
+        {
+            _db = context;
+            _config = config;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Memberships>>> Get()
```

Comment:
Its better to give classes singular names, `Memberships` -> `Membership`, otherwise the table name created by Entity Framework might be weird

Reply:
I was advised to use this way, to change would be quiet taxing, is the naming convention that poor that it has to be changed?

Reply:
I see that in the migration file almost everything is memberships, dunno if it would be that quick to refactor all of it. But @DreyerChris, you do make sense. Your call

Reply:
@RetroSabian Check what happens to the migrations if you change it. Yes it is really bad practice to give classes plural names. A `Membership` object is one instance of the class, therefore it should not have a plural name.

Reply:
@RetroSabian Test it first in a separate branch to check if it breaks the migration. Then you can decide if you want to leave it or if you want to include it and manually fix the migrations

Reply:
HI Chris, i redid the whole database, pease check it out